mungegithub/mungers/release-note-label_test.go

```diff
@@ -249,6 +249,11 @@ func TestGetReleaseNote(t *testing.T) {
 			expectedReleaseNoteVariable: releaseNoteNone,
 		},
 		{
+			body:                        "**Release note**:\n\n<other unimportant information> ```NONE```",
```

Comment:
We ask specifically n the release-note template for the release note to be between three ticks.
E.g. ```Release Note Goes Here```